Makefile

```diff
@@ -143,11 +143,15 @@ ADDONS_BINDING_GYPS := \
 	$(filter-out test/addons/??_*/binding.gyp, \
 		$(wildcard test/addons/*/binding.gyp))
 
+ADDONS_BINDING_SOURCES := \
+	$(filter-out test/addons/??_*/*.cc, $(wildcard test/addons/*/*.cc)) \
+	$(filter-out test/addons/??_*/*.h, $(wildcard test/addons/*/*.h)) \
```

Comment:
tiny nit: the trailing `\` here is superfluous/inconsistent with the way line continuations are ended in the rest of the Makefile